rental.rb

```diff
@@ -0,0 +1,25 @@
+require './person'
+require './book'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] `book` and `person` should have setters and getters. Please remove them from the `attr_reader` and define them in the `attr_accessor`
This will enable setting and reading easy.